client/src/app/pages/questionnaires/Questionnaires.tsx

```diff
@@ -0,0 +1,478 @@
+                                  <EllipsisVIcon />
+                                </MenuToggle>
+                              )}
+                              shouldFocusToggleOnSelect
+                            >
+                              <DropdownItem
+                                key="export"
+                                component="button"
+                                onClick={() =>
+                                  setQuestionnaireToExport(questionnaire.id)
+                                }
+                              >
+                                {t("actions.export")}
+                              </DropdownItem>
+                              <DropdownItem
+                                key="view"
+                                component="button"
+                                onClick={() => {
+                                  // TODO Link to questionnaire page
+                                  // history.push(Paths.questionnaire);
+                                }}
+                              >
+                                {t("actions.view")}
+                              </DropdownItem>
+                              <ConditionalTooltip
+                                key="system-questionnaire"
+                                isTooltipEnabled={questionnaire.system}
+                                content={
+                                  "Disabled because it is a system questionnaire."
+                                }
+                              >
+                                <DropdownItem
+                                  key="delete"
+                                  isAriaDisabled={questionnaire.system === true}
+                                  onClick={() =>
+                                    setQuestionnaireToDelete(questionnaire.id)
+                                  }
+                                >
+                                  {t("actions.delete")}
+                                </DropdownItem>
+                              </ConditionalTooltip>
+                            </Dropdown>
+                          </Td>
+                        </TableRowContentWithControls>
+                      </Tr>
+                    </Tbody>
+                  );
+                })}
+              </ConditionalTableBody>
+            </Table>
+            <SimplePagination
+              idPrefix="questionnaires-table"
+              isTop={false}
+              paginationProps={paginationProps}
+            />
+          </div>
+        </ConditionalRender>
+      </PageSection>
+      <Modal
+        id="import.modal"
+        title={t("dialog.title.import", {
+          what: t("terms.questionnaire").toLowerCase(),
+        })}
+        variant={ModalVariant.medium}
+        isOpen={isImportModal}
+        onClose={() => setIsImportModal(false)}
+      >
+        <Text>TODO Import questionnaire component</Text>
+      </Modal>{" "}
```

Comment:
The `{" "}`s here and on line 440 can probably be removed since Modal doesn't render anything inline, so there's no need for space characters between them